### PR TITLE
Introduce test_user_stories.py

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**

### CODE REVIEW

File: proxy/deploy-test.sh [hunk 1]
```diff
@@ -3,91 +3,8 @@ set -xeuo pipefail
 
 echo "Deploy test..."
 
-curl -v --header "Content-Type: application/json" --data '{"method":"eth_blockNumber","id":1001,"jsonrpc":"2.0","params":[]}' $PROXY_URL
+curl -v --header "Content-Type: application/json" --data '{"method":"eth_blockNumber","id":1,"jsonrpc":"2.0","params":[]}' $PROXY_URL
```
Comment (s-medvedev) [inline proxy/deploy-test.sh:6]: Why did you leave this request?

File: proxy/test_user_stories.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
...1840152602081019050610068565b50505050905090810190601f1680156100b05780820380516001836020036101000a031916815260200191505b509250505060405180910390f35b6100c66101df565b6040518080602001828103825283818151815260200191508051906020019080838360005b838110156101065780820151818401526020810190506100eb565b50505050905090810190601f1680156101335780820380516001836020036101000a031916815260200191505b509250505060405180910390f35b60008054600181600116156101000203166002900480601f0160208091040260200160405190810160405280929190818152602001828054600181600116156101000203166002900480156101d75780601f106101ac576101008083540402835291602001916101d7565b820191906000526020600020905b8154815290600101906020018083116101ba57829003601f168201915b505050505081565b606060008054600181600116156101000203166002900480601f0160208091040260200160405190810160405280929190818152602001828054600181600116156101000203166002900480156102775780601f1061024c57610100808354040283529160200191610277565b820191906000526020600020905b81548152906001019060200180831161025a57829003601f168201915b505050505090509056fea265627a7a7231582024368df40ce2133f972294ddde9f574e801391af7268266abe1646f640b2294c64736f6c63430005110032"}]}' https://proxy.testnet.neonlabs.org/solana
```
Comment (s-medvedev) [inline proxy/test_user_stories.py]: Specifying the curl request in the comment seems like duplicate code. Maybe it should be removed? All the information you need is in the arguments of `request.post` call.

**Review by s-medvedev — DISMISSED** [review]

File: proxy/deploy-test.sh (hunk 1 above)
Comment (vasiliy-zaznobin) [reply proxy/deploy-test.sh:6]: This request has always been, it checks the connection with the proxy

File: proxy/test_user_stories.py (hunk 2 above)
Comment (vasiliy-zaznobin) [reply proxy/test_user_stories.py]: Fixed

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**